deltachat-ios/AccountSetupController.swift

```diff
@@ -222,18 +221,17 @@ class AccountSetupController: UITableViewController {
     if section == 0 {
       // basicSection
       return basicSectionCells[row]
-		} else if section == 1 {
-			return restoreCells[row]
+    } else if section == 1 {
+      return restoreCells[row]
     } else {
       // advancedSection
       return advancedSectionCells[row]
     }
   }
 
   override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-
-		guard let tappedCell = tableView.cellForRow(at: indexPath) else { return }
-		// handle tap on password -> show oAuthDialogue
+    guard let tappedCell = tableView.cellForRow(at: indexPath) else { return }
+    // handle tap on password -> show oAuthDialogue
     if let textFieldCell = tappedCell as? TextFieldCell {
       if textFieldCell.accessibilityIdentifier == "passwordCell" {
         if let emailAdress = textFieldCell.getText() {
```

Comment:
seems to me like unnecessary nesting of if conditions instead of using &&, but it's rather a code style issue from my point of view

Reply:
I know looks awful.

Explanation:
237 and 239 are actually for unwrapping optionals, so you are assigning a value to a variable that is guaranteed to be not null. 
238 is a conditional.
Both statements can't be combined unfortunately.